playwright/_impl/_browser_type.py

```diff
@@ -187,27 +190,31 @@ async def connect(
         transport = WebSocketTransport(
             self._connection._loop, ws_endpoint, headers, slow_mo
         )
+        playwright_future: asyncio.Future = asyncio.Future()
```

Comment:
```suggestion
        playwright_future: asyncio.Future["Playwright"] = asyncio.Future()
```

Reply:
we don't use playwright_future any further, so it does not bring any value.

Reply:
It is used here 
```py
        async def handle_ready() -> None:
            playwright_future.set_result(await connection.initialize_playwright())
```